tests/peepopt/multdiv_c.ys

```diff
@@ -0,0 +1,49 @@
+log -header "Test simple positive case"
+log -push
+design -reset
+read_verilog <<EOF
+module top (
+  input wire  [31:0] a,
+  output wire  [63:0] y
```

Comment:
proving equivalence of large multipliers with a plain sat solver is generally gonna be slow so if possible id suggest here just using slightly smaller bitwidths, or maybe consider putting these two tests in separate test files that can be run in parallel.

Reply:
@georgerennie This case takes <30 seconds as-is, I'll reduce the size.

Reply:
10-30 seconds is extreme. If you can exercise the same codepaths in 300ms instead by reducing the bit widths, there's no reason not to do so. Or you can duplicate the test and then only run formal on the easy version and only check an effect of doing/not doing the transform in the hard version. I didn't find any issues stretching the pass with 4 bit multiplication and 8 bit division

Reply:
Fixed, below a second for each test (2 in total).
I also changed back to equiv_opt as this transform keep the logical wires intact and equiv_opt indeed works here.